embedding-store-filter-parsers/langchain4j-embedding-store-filter-parser-sql/src/main/java/dev/langchain4j/store/embedding/filter/builder/sql/LanguageModelSqlFilterBuilder.java

```diff
@@ -88,39 +87,41 @@ public class LanguageModelSqlFilterBuilder {
     private static final Logger log = LoggerFactory.getLogger(LanguageModelSqlFilterBuilder.class);
 
     private static final PromptTemplate DEFAULT_PROMPT_TEMPLATE = PromptTemplate.from(
-            "### Instructions:\n" +
-                    "Your task is to convert a question into a SQL query, given a Postgres database schema.\n" +
-                    "Adhere to these rules:\n" +
-                    "- **Deliberately go through the question and database schema word by word** to appropriately answer the question\n" +
-                    "- **Use Table Aliases** to prevent ambiguity. For example, `SELECT table1.col1, table2.col1 FROM table1 JOIN table2 ON table1.id = table2.id`.\n" +
-                    "- When creating a ratio, always cast the numerator as float\n" +
-                    "\n" +
-                    "### Input:\n" +
-                    "Generate a SQL query that answers the question `{{query}}`.\n" +
-                    "This query will run on a database whose schema is represented in this string:\n" +
-                    "{{create_table_statement}}\n" +
-                    "\n" +
-                    "### Response:\n" +
-                    "Based on your instructions, here is the SQL query I have generated to answer the question `{{query}}`:\n" +
-                    "```sql"
-    );
+            """
+            ### Instructions:
```

Comment:
Could you please include only changes to the prompt? Reformatting in scope of this PR makes it very hard to see the actual changes